### DISCUSSION TITLE
FEATURE: electron的版本什么时候出？

### DISCUSSION THREAD

**Discussion by KuanyaLiao** [vbenjs/vue-vben-admin#4022] [General]
### Version

Vben Admin Pro

### Description

希望出个electron的版本

### Proposed Solution

希望出个electron的版本

### Validations

- [X] Read the [docs](https://anncwb.github.io/vue-vben-admin-doc/)
- [X] Ensure the code is up to date. (Some issues have been fixed in the latest version)
- [X] I have searched the [existing issues](https://github.com/vbenjs/vue-vben-admin/issues) and checked that my issue does not duplicate any existing issues.

**Comment by jinmao88** [comment]
不好说

**Comment by canyanol650** [comment]
我基于最新版 v5 做了 electron 集成. https://github.com/canyanol650/vue-vben-admin-electron.
npm run electron:dev --开发
npm run electron:build -- 构建
